src/styles/styles.js

```diff
@@ -37,17 +37,21 @@ const styles = {
         marginLeft: 8,
     },
 
-    mt2: {
-        marginTop: 20,
-    },
-
     mt1: {
         marginTop: 4,
     },
-
+    mt2: {
+        marginTop: 8,
+    },
```

Comment:
NAB, but we should add some consistency to the values used for these. If we are incrementing these helpers by `4` that's cool but we should follow up and do this everywhere.

Reply:
We do mostly do this everywhere already, but yes, we should be consistent and I fixed a bit of the inconsistency here.